### ISSUE TITLE
Cursor position after pressing verify

### ISSUE THREAD

**Issue by dreams-togo** [FreeUKGen/ComETT#81]
After pressing verify it would be helpfull in the next screen for the next record if the cusor selects the first field in the new record (ofter the register no. in post 1812 records)

**Comment by stsccfr** [comment]
Yes, positioning the cursor in the first data field after entering a record would help speed up transcription. This would be very desirable.  (Note that pressing Verify is now irrelevant since Verification has been dropped.)

**Comment by stsccfr** [comment]
After entering a record, the cursor is now being placed in the first data entry field, so this case can be closed.
